mapeditor/generatorprogress.cpp

```diff
@@ -19,11 +19,11 @@ GeneratorProgress::GeneratorProgress(Load::Progress & source, QWidget *parent) :
 	source(source)
 {
 	ui->setupUi(this);
+	
+	setWindowFlags(Qt::Dialog | Qt::WindowTitleHint | Qt::CustomizeWindowHint);
```

Comment:
are you trying to remove the "question mark" button on Windows for all dialogs? There's an easier way:
```cpp
setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
```

Reply:
Yes, that works too. But I have choosed full flags control instead. So instead of just removing "?" help hutton. I'm setting what exactly should be there - Title + Close button and overwrite all flags there. There are probably no flags set.

Reply:
And for this generatorprogress is not only about "?" help button removal. It's also about "x" close button removal to never allow cancelling map generating which will lead into crash.